src/app/sign-in/pages/sign-in/sign-in.component.ts

```diff
@@ -26,7 +26,6 @@ export class SignInComponent implements OnInit {
   @ViewChild('formSignInComponent') formSignInComponent: FormSignInComponent
   oauthParameters: OauthParameters
   requestInfoForm: RequestInfoForm
-  signInLocal = {} as SignInLocal
```

Comment:
@DanielPalafox Please take a careful look here, I did remove this signInLocal since the variable was never assigned. 

With that, I also remove a conditional and a method that was dependent on this been assign